patches/master_patch.patch

```diff
@@ -1010,6 +1010,42 @@ index 1c68e13912c2dfa7cc842bab169029c58677811a..3731fe3df76b0ac1dbf155c221fc64e2
  }
  
  content::WebContents* GuestViewManager::GetGuestByInstanceIDSafely(
+diff --git a/components/history/core/browser/history_service.cc b/components/history/core/browser/history_service.cc
```

Comment:
I think it's a race condition because the task tracker is on the stack. I think it should work if we make it an instance var on atom_api_session